boards/arm/pan1783_evb/pan1783_evb_cpunet_reset.c

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2023 Panasonic Industrial Devices Europe GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/init.h>
+#include <zephyr/logging/log.h>
+
+#include <soc.h>
+#include <hal/nrf_reset.h>
+
+LOG_MODULE_REGISTER(pan1783_evb_cpuapp, CONFIG_LOG_DEFAULT_LEVEL);
+
+#if defined(CONFIG_BT_CTLR_DEBUG_PINS_CPUAPP)
+#include <../subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/debug.h>
+#else
+#define DEBUG_SETUP()
+#endif
+
+static void remoteproc_mgr_config(void)
+{
+#if !defined(CONFIG_TRUSTED_EXECUTION_NONSECURE) || defined(CONFIG_BUILD_WITH_TFM)
```

Comment:
There is no secure/non-secure split for this board so conditions like this one and related comments in this file are a bit confusing.